parlai/tasks/coqa/build.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+# Download and build the data if it does not exist.
+
+import parlai.core.build_data as build_data
+import os
+import json
+
+TRAIN_FILENAME = 'coqa-train-v1.0.json'
+VALID_FILENAME = 'coqa-dev-v1.0.json'
+URL = 'https://nlp.stanford.edu/data/coqa/'
+
+
+def make_parlai_format(outpath, dtype, data):
+    print('building parlai:' + dtype)
+    for each in data:
+        output = []
+        story = each['story'].replace('\n', '\\n')
+        for question, ans in zip(each['questions'], each['answers']):
+            question_txt = ''
+            if question['turn_id'] == 1:
+                # include the context in the very first turn
+                question_txt = story + '\\n' + question['input_text']
+            else:
+                question_txt = question['input_text']
+            output.append("text:{question}\tlabels:{labels}".format(
+                question=question_txt,
+                labels=ans['input_text'].replace("|", " __PIPE__ ")
```

Comment:
Watch those double quotes.

Reply:
Sorry for that, I thought that was a specific usage from your comments. Will fix that and add copyright info in __init__.py then merge.